rollup.config.mjs

```diff
@@ -48,53 +42,97 @@ const node_build = {
       exports: 'named',
     },
   ],
+  external: ['debug'],
   plugins: [
     // Locate modules using the Node resolution algorithm, for using third party modules in node_modules
     nodeResolve({
-      browser: true, // Only resolve browser-compatible modules
+      browser: false, // "browser" properties in package files are ignored
       preferBuiltins: true, // Prefer built-in modules
     }),
     // Convert CommonJS modules to ES6, so they can be included in a Rollup bundle
     commonjs(),
-    // Transpile
+    // Transpile TypeScript to JavaScript
```

Comment:
Looks like AI comments :p

Reply:
Hahaha, I asked to improve the doc a little cause I don't trust my Enlish 100%, looks like this slipped :smile: